src/scripts/toc-utils.ts

```
@@ -0,0 +1,105 @@
+}
+
+/**
+ * Filtre les headings par profondeur maximale
+ * @param headings - Array des headings
+ * @param maxDepth - Profondeur maximale à inclure
+ * @returns Headings filtrés
+ */
+export function filterHeadingsByDepth(
+  headings: AstroHeading[], 
+  maxDepth: number = 4
+): AstroHeading[] {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)